install/tests/Makefile

```diff
@@ -277,6 +296,9 @@ generate-kots-config: select-workspace check-env-cloud get-base-config
 	$(MAKE) storage-config-${cloud_storage}
 	$(MAKE) db-config-${cloud_db}
 	$(MAKE) registry-config-${cloud_registry}
+ifndef self_signed
```

Comment:
Same as above, `ifndef` passes if the variable is not defined

Reply:
you can do something like `[[ -z "$$self_signed" ]] && $(MAKE) self-signed-config` (need testing)